custom_model_runner/datarobot_drum/drum/description.py

```diff
@@ -4,6 +4,6 @@
 This is proprietary source code of DataRobot, Inc. and its affiliates.
 Released under the terms of DataRobot Tool and Utility Agreement.
 """
-version = "1.16.0"
+version = "1.16.1"
```

Comment:
Will you update `custom_model_runner/CHANGELOG.md` for this new minor version changes?